### PR TITLE
WIP: Trying limiting the number of OpenBLAS threads

### PR DESCRIPTION
**Pull request by patricksnape**
Since the new release of OpenBLAS now has threading enabled,
perhaps this is causing issues on circleci with suitesparse?

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ test:
     - sksparse
     - sksparse.cholmod
   commands:
-    - nosetests -sv sksparse
+    - OPENBLAS_NUM_THREADS=1 nosetests -sv sksparse
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Running the tests with `OPENBLAS_NUM_THREADS=1` will still ship a broken package if the multi-threaded `openblas` is an issue. I remember a talk at SciPy mentioned that many bugs with multi-threaded openblas were considered "features" and would never be fixed :unamused:

Comment (patricksnape) [inline recipe/meta.yaml]: This is only an issue on CircleCI. It runs totally fine on my machine - its just hanging on CircleCI

Comment (ocefpaf) [inline recipe/meta.yaml]: Do you know why it is an issue on `CircleCI`?

Comment (patricksnape) [inline recipe/meta.yaml]: I don't know but I just reverted the commit and it timed-out again!

**Comment by patricksnape** [comment]
@grlee77 looks like we need to set the OpenBLAS threads.

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipe/meta.yaml]: Just curious... I abolished `openblas` from my local builds due to similar time-out errors and some bad matrix manipulation results. When I asked upstream I got a "this is a feature and not a bug." So I decided to stick with `mkl` after that.

Comment (patricksnape) [inline recipe/meta.yaml]: I thought we couldn't really use MKL because it doesn't ship with the headers/all the dynamic libraries?

Comment (ocefpaf) [inline recipe/meta.yaml]: I am relying on `defaults` packages for that.

**Comment by grlee77** [comment]
LGTM and the tests pass now, but I am a little concerned the package may potentially be broken on some systems.  I also cannot reproduce the issue when building locally on a linux workstation.

Can you bump the build number to 1?  Also, if interested, please add yourself as a maintainer.

**Comment by jakirkham** [comment]
The current issue can be fixed with a re-rendering as it is unrelated to the package per se.

**Comment by patricksnape** [comment]
I tried installing latest conda-smithy and rerendering fails...

**Comment by jakirkham** [comment]
How did you try to install it?

**Comment by patricksnape** [comment]
It just wasn't working on my Linux box but I'm wondering if I had it `pip` installed as well or something. Anyway, worked on my OSX laptop.

**Comment by patricksnape** [comment]
@grlee77 passes now.

**Comment by grlee77** [comment]
Great!  thanks

**Pull request merged**